README.md

```diff
@@ -96,6 +96,23 @@ After that you can login to DataLens on http://localhost:8080 using the default
 
 You can use the same credentials to configure Zitadel and add new users using Zitadel control panel at http://localhost:8085/. **Don't forget to login there at least once to change the default password.**
 
+### Configuring Zitadel on external domain
+
+If your Zitadel is going to be used on external domain like http://your-domain.com:8085 instead of http://localhost:8085 (by default) you need to do the following steps:
+
+1. After running `init.sh` script change this setting in `docker-compose.zitadel.yml` file to your domain:
+
+```bash 
+ZITADEL_URI: "http://your-domain.com:8085"
+```
+
+2. Run the following command to setup Zitadel at your domain (you need to run this command at least once):
+
+```bash 
+ZITADEL_EXTERNALPORT=8085 ZITADEL_EXTERNALDOMAIN=your-domain.com docker compose -f docker-compose.zitadel.yml up zitadel -d
```

Comment:
Will this work if we've already run zitadel without these parameters before?

Reply:
Yes, this command teaches Zitadel to trust this external domain

Reply:
1. Looks like we're passing same information two times: ZITADEL_URI vs ZITADEL_EXTERNALDOMAIN & ZITADEL_EXTERNALPORT, can we specify it once somewhere?
2. Can I somehow specify external domain **before** running init.sh so it will be configured correctly right away without the need for additonal steps?

Reply:
Both your ideas are great and can be done, but I also have idea to make init.sh interactive with users (asking to specify these values while running init.sh). Let's discuss offline pros and cons of both approaches and decide how to proceed further.

Reply:
Discussed offline with @resure & @goshander. Decided to pass Zitadel external domain as a param to init.sh. It will be done for on-prem version first (within a couple of days) and then used the same approach for oss.